README.md

```diff
@@ -349,15 +349,29 @@ For Windows, run:
 
 ## Sponsors
 
-Livebook development is sponsored by:
+<a href="https://fly.io">
+  <img height="175" src="https://fly.io/public/images/brand/logo.svg" alt="Fly.io">
+</a>
 
-<a href="https://fly.io" target=_blank><img src="https://fly.io/public/images/brand/logo.svg" width="320" /></a>
+Fly is a platform for running full stack apps and databases close to your users.
 
-## Supporters
+<br />
+<br />
 
-Machine Learning and Neural Network models hosted by:
+<a href="https://huggingface.co/">
+  <img height="100" src="https://huggingface.co/datasets/huggingface/brand-assets/resolve/main/hf-logo-with-title.png" alt="Hugging Face">
+</a>
 
-<a href="https://huggingface.co/" target=_blank><img src="https://huggingface.co/datasets/huggingface/brand-assets/resolve/main/hf-logo-with-title.png" width="320" /></a>
+Machine Learning and Neural Network models hosted by Hugging Face.
```

Comment:
I think we should go with Hugging Face official description here too, wdyt?

Reply:
Done. ✅